### PR TITLE
LPS-0 DO NOT MERGE Run all forms tests

### PR DESCRIPTION
**Pull request by clovisdasilvaneto**

### CODE REVIEW

**Comment by clovisdasilvaneto** [comment]
ci:test

**Comment by alinedoleron** [comment]
LGTM.

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-editor/src/main/resources/META-INF/resources/editor.soy [hunk 1]
```diff
@@ -16,9 +16,9 @@
 	{@param placeholder: string}
 	{@param required: bool}
 	{@param showLabel: bool}
-	{@param? tip: string}
 	{@param value: string}
 	{@param visible: bool}
+	{@param? tip: string}
```
Comment (diegonvs) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-editor/src/main/resources/META-INF/resources/editor.soy]: Hey @clovisdasilvaneto , could you put these declarations in alphabetical order?

Comment (clovisdasilvaneto) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-editor/src/main/resources/META-INF/resources/editor.soy]: @diegonvs according the check_sf, optional parameters should be at the end

**Comment by clovisdasilvaneto** [comment]
ci:retest

**Comment by rafaprax** [comment]
@rodrigo-e-c-souza Could you please check if these errors are related to @clovisdasilvaneto changes? Thx

**Comment by rodrigo-e-c-souza** [comment]
ci:retest

**Comment by rodrigo-e-c-souza** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by rodrigo-e-c-souza** [comment]
LGTM!

**Comment by pedroqueiroz94** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by pedroqueiroz94** [comment]
Sent at https://github.com/brianchandotcom/liferay-portal/pull/54906.

**Pull request closed without merging**